cmd/agent/app/agent_test.go

```diff
@@ -51,6 +94,10 @@ func TestAgentStartStop(t *testing.T) {
 		HTTPServer: HTTPServerConfiguration{
 			HostPort: ":0",
 		},
+		Metrics: jmetrics.Builder{
+			Backend:   "prometheus",
+			HTTPRoute: "/metrics",
```

Comment:
I had to place the default values here as they are usually initialized using Viper